docs/topics/class-based-views/intro.txt

```diff
@@ -173,29 +173,6 @@ that inherits from ``View`` - for example, trying to use a form at the top of a
 list and combining :class:`~django.views.generic.edit.ProcessFormView` and
 :class:`~django.views.generic.list.ListView` - won't work as expected.
 
-.. _mixins_that_wrap_as_view:
-
```

Comment:
If we can come up with another use case to showcase the behavior, I'm all in. Otherwise I'd drop it for now. There is another example that decorates `dispatch()` by using the `login_required` decorator. And I'm all in favor of decorating `dispatch()` over `as_view()`. It's too easy to forget `@classdecorator` or `@classonlydecorator`.